test/specs/components/action.spec.jsx

```diff
@@ -195,19 +221,24 @@ xdescribe('Action', () => {
         });
     });
 
-    describe('buy action without stripe keys', () => {
+    // Re-visit this test after fixing link fallback
```

Comment:
We used to check for keys, but that got removed at some point: https://github.com/smooch/smooch-js/commit/e8bb52380f3ff23ff8fa589d21ee4157b2881c2d#diff-1176353b9d32bfd3b90e28ff3100b43cL90